File: src/main/java/com/wgzhao/fsbrowser/model/pg/VwAddaxLog.java
package com.wgzhao.fsbrowser.model.pg;

import jakarta.persistence.Basic;
import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.Getter;
import lombok.Setter;

import java.util.Objects;

@Entity
@Table(name = "vw_addax_log", schema = "public", catalog = "stg01")
@Setter
@Getter
public class VwAddaxLog {

    @Id
    @Basic
    @Column(name = "start_day")
    private String startDay;
    
    @Basic
    @Column(name = "spname")
    private String spname;
    @Basic
    @Column(name = "start_time")
    private String startTime;
    @Basic
    @Column(name = "end_time")
    private String endTime;
    @Basic
    @Column(name = "runtime")
    private Integer runtime;
    @Basic
    @Column(name = "byte_speed")
    private Integer byteSpeed;
    @Basic
    @Column(name = "rec_speed")
    private Integer recSpeed;
    @Basic
    @Column(name = "total_rec")
    private Integer totalRec;
    @Basic
    @Column(name = "total_err")
    private Integer totalErr;
}

File: src/main/kotlin/com/wgzhao/addax/admin/model/VwAddaxLog.kt
package com.wgzhao.addax.admin.model

import jakarta.persistence.*

@Entity
@Table(name = "vw_addax_log", schema = "public", catalog = "stg01")
data class VwAddaxLog(
    @Id
    @Basic
    @Column(name = "start_day")
    var startDay: String? = null,

    @Basic
    @Column(name = "spname")
    var spname: String? = null,

    @Basic
    @Column(name = "start_time")
    var startTime: String? = null,

    @Basic
    @Column(name = "end_time")
    var endTime: String? = null,

    @Basic
    @Column(name = "runtime")
    var runtime: Int? = null,

    @Basic
    @Column(name = "byte_speed")
    var byteSpeed: Int? = null,

    @Basic
    @Column(name = "rec_speed")
    var recSpeed: Int? = null,

    @Basic
    @Column(name = "total_rec")
    var totalRec: Int? = null,

    @Basic
    @Column(name = "total_err")
    var totalErr: Int? = null
)
